src/components/Page/Page.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Title from "../Title/Title"
+import CardContainer from '../CardContainer/CardContainer'
+
+function Page() {
+    return (
+        <div>
```

Comment:
Are we wrapping the components in a div so that we aren't passing down sibling elements? If so, we can use React Fragments, and then the unnecessary div isn't rendered. Ignore this comment if you do actually need that div though
 -- this was a comment on the previous review but was marked as resolved with no comments or change

Reply:
on it